### PR TITLE
TC_02.08.004 | Dashboard > verify "Configure a cloud" Link

### PR DESCRIPTION
**Pull request by mrsKseniaKotova**
https://trello.com/c/8q1voF1I/281-us0208004-dashboard-block-set-up-a-distributed-build-configure-a-cloud-opens

### CODE REVIEW

File: cypress/e2e/openConfigureCloud.cy.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress"/>
+
+describe('open Configure a Cloud', () => {
+    it('US_02.08.004 | Dashboard -> Block “Set up a distributed build” -> "Configure a cloud" opens', () => {
+        cy.get('#main-panel > div:nth-child(3) > div > section:nth-child(4) > h2')
+            .should('have.text', 'Set up a distributed build');
```
Comment (S-Lanchik) [inline cypress/e2e/openConfigureCloud.cy.js:6]: Put 'Set up a distributed build' to fixtures

File: cypress/e2e/openConfigureCloud.cy.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  
+        cy.get('a[href="computer/new"]').should('be.visible');
+        cy.get('a[href="cloud/"]').should('be.visible');
+        cy.get('a[href="https://www.jenkins.io/redirect/distributed-builds"]').should('be.visible');
+
+        cy.get('a[href="cloud/"]').click();
+
+        cy.get('.jenkins-app-bar__content').should('have.text', 'Clouds');
```
Comment (S-Lanchik) [inline cypress/e2e/openConfigureCloud.cy.js:14]: put 'Clouds" to fixtures

**Comment by S-Lanchik** [comment]
Please edit the PR name to be as your test name

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/openConfigureCloud.cy.js (hunk 2 above)
Comment (mrsKseniaKotova) [reply cypress/e2e/openConfigureCloud.cy.js]: Thank you. Done. Could you check again, please?

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**